### PR TITLE
Add sdk and cli example for binary payloads

### PR DESCRIPTION
**Pull request by xanwal**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Add sdk and cli example for binary payloads

fixes #

### CODE REVIEW

File: cli/endpoints/online/managed/binary-payloads/code/multi-file-to-json-score.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from azureml.contrib.services.aml_request import AMLRequest, rawhttp
```
Comment (shohei1029) [inline cli/endpoints/online/managed/binary-payloads/code/multi-file-to-json-score.py:1]: Is this package from SDK v1?
Do we have any ways to do this with SDK v2?

File: cli/endpoints/online/managed/binary-payloads/code/single-file-to-file-score.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from azureml.contrib.services.aml_request import AMLRequest, rawhttp
+from azureml.contrib.services.aml_response import AMLResponse
```
Comment (shohei1029) [inline cli/endpoints/online/managed/binary-payloads/code/single-file-to-file-score.py:2]: Ditto

**Review by shohei1029 — CHANGES_REQUESTED** [review]

**Comment by shohei1029** [comment]
added @ahughes-msft as an assignee.

**Comment by xanwal** [comment]
CLI and SDK versions passing on commit https://github.com/Azure/azureml-examples/pull/1795/commits/2058da267f742e6a20069c36e0f083b063a49830

File: cli/endpoints/online/managed/binary-payloads/code/multi-file-to-json-score.py (hunk 1 above)
Comment (xanwal) [reply cli/endpoints/online/managed/binary-payloads/code/multi-file-to-json-score.py:1]: Last thing here, theoretically an easy change but I had some problems with this recently.

File: cli/endpoints/online/managed/binary-payloads/code/single-file-to-file-score.py (hunk 2 above)
Comment (xanwal) [reply cli/endpoints/online/managed/binary-payloads/code/single-file-to-file-score.py:2]: It doesn't seem like there's a way to get rid of these, although the user doesn't actually have to have the package installed.

File: cli/deploy-moe-binary-payloads.sh [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+# <set_variables> 
+ENDPOINT_NAME=endpt-moe-`echo $RANDOM`
+ACR_NAME=$(az ml workspace show --query container_registry -o tsv | cut -d'/' -f9-)
+# </set_variables>
+
+BASE_PATH="endpoints/online/managed/binary-payloads"
+
+# <download_sample_data> 
+wget https://aka.ms/peacock-pic -O binary-payloads/input.jpg
```
Comment (shohei1029) [inline cli/deploy-moe-binary-payloads.sh:12]: The runner failed here...

**Review by dem108 — APPROVED** [review]
Great addition. Thanks!!

**Review by shohei1029 — APPROVED** [review]

**Pull request merged**